oslo/torch/nn/modules/layer_norm.py

```diff
@@ -89,6 +90,28 @@ def __init__(
             dtype=dtype,
         )
 
+    def forward(self, input: Tensor) -> Tensor:
```

Comment:
Is this only for tpsp?

Reply:
If we use tp1d, we always use it. (if not use tpsp, it is the same as normal LayerNorm1D though)

Reply:
ah there were if-else conditions. I understood.